service.subtitles.traduttorianonimi/service.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+#
+#  service.py
+#  
+#  Copyright 2016 ShellAddicted <shelladdicted@gmail.com<>
+#  
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License
+#  
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+#  MA 02110-1301, USA.
+#  
+#
+import os;
+import sys;
+import unicodedata;
+import xbmc;
+import xbmcvfs;
+import xbmcaddon;
+import xbmcgui;
+import xbmcplugin;
+
+import urllib;
+import requests;
+import zipfile;
+import StringIO;
+import traceback;
+
+__addon__ = xbmcaddon.Addon();
+__author__ = __addon__.getAddonInfo("author");
+__scriptid__ = __addon__.getAddonInfo("id");
+__scriptname__ = __addon__.getAddonInfo("name");
+__version__ = __addon__.getAddonInfo("version");
+__language__ = __addon__.getLocalizedString;
+main_url = "http://traduttorianonimi.it";
+
+__cwd__ = xbmc.translatePath(__addon__.getAddonInfo("path")).decode("utf-8");
```

Comment:
In your next update please remove the \__varname\__ style labels. These should only be used for python internals.